calico_node/startup/startup.go

```diff
@@ -749,19 +761,19 @@ func ensureGlobalBGPConfig(c *client.Client, key, def string) error {
 // message prints a message to screen and to log.  A newline terminator is
 // not required in the format string.
 func message(format string, args ...interface{}) {
-	fmt.Printf(format+"\n", args...)
+	log.Infof(calicoctlLogPrefix+format, args...)
 }
 
 // warning prints a warning to screen and to log.  A newline terminator is
 // not required in the format string.
 func warning(format string, args ...interface{}) {
-	fmt.Printf("WARNING: "+format+"\n", args...)
+	log.Warningf(calicoctlLogPrefix+"WARNING: "+format, args...)
```

Comment:
Do we still want the `WARNING` bit produced here? I would have thought calicoctl could add that if it wants to, but it's not necessary here now that we're using `log.Warningf`